tests/omhttp-basic.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+# This file is part of the rsyslog project, released under ASL 2.0
+
+#  Starting actual testbench
+. ${srcdir:=.}/diag.sh init
+
+port="$(get_free_port)"
+omhttp_start_server $port
+
+error_file=$(pwd)/$RSYSLOG_DYNNAME.omhttp.error.log
+rm -f $error_file
+
+generate_conf
+add_conf '
+#$DebugLevel 2
+template(name="tpl" type="string"
+	 string="{\"msgnum\":\"%msg:F,58:2%\"}")
+
+module(load="../contrib/omhttp/.libs/omhttp")
+
+if $msg contains "msgnum:" then
+	action(
+		# Payload
+		name="my_http_action"
+		type="omhttp"
+		errorfile="'$error_file'"
+		template="tpl"
+
+		server="localhost"
+		serverport="'$port'"
+		restpath="my/endpoint"
+		batch="off"
+
+		# Auth
+		usehttps="off"
+    )
+'
+startup
+injectmsg  0 10000
```

Comment:
if you use
```
export NUMMESSAGES=10000
```
after test init, you do not need to specify the count in ìnjectmsg`and `seq_check` - makes it a bit easier to change the count (for whatever reason).